api/app/src/test/kotlin/packit/integration/services/OutpackServerClientTest.kt

```diff
@@ -24,4 +25,16 @@ class OutpackServerClientTest : IntegrationTest()
         val result = sut.getMetadata()
         assert(result.map { it.name }.containsAll(listOf("parameters", "explicit", "depends", "computed-resource")))
     }
+
+    @Test
+    fun `can write the input stream from a get response to an output stream`()
+    {
+        val hash = "sha256:1a1b649d911106d45dcb58e535aae97904f465e66b11a38d7a70828b53e3a2eb"
```

Comment:
Might be nice to include a comment here saying what file from what packet you're expecting this to be!